src/Compilers/Core/Portable/Symbols/Nullability.cs

```diff
@@ -79,6 +79,13 @@ public enum Nullability
         /// </remarks>
         NotApplicable = 0,
         /// <summary>
+        /// The given <see cref="SyntaxNode"/> has inferred nullability that has not yet been
+        /// computed.
+        /// </summary>
+        // PROTOTYPE(NullableReferenceTypes): Potentially link to the correct methods to compute?
+        // Some of them are defined on CSharpSemanticModel, so this could be difficult
+        NotComputed = 1,
```

Comment:
exposing this publicly seems... weird.  does that mean users can observe when we have haven't comptued this stuff?